include/irr/asset/CTexturePacker.h

```diff
@@ -60,21 +121,13 @@ class CTexturePacker
         auto extent = _img->getCreationParameters().extent;
         uint32_t rnd = core::roundUpToPoT(std::max(extent.width,extent.height))/m_pgSzxy;
         uint32_t pgCnt = rnd*rnd;
```

Comment:
nope, convert texture size to page extents while rounding up, THEN round Up to Power of Two (but the page size)
```cpp
uint32_t pageSize[2];
pageSize[0] = (extent.width+m_pgSz[0]-1u)/m_pgSz[0];
pageSize[1] = (extent.height+m_pgSz[1]-1u)/m_pgSz[1];

// then obtain one-past the end morton code but in pages.
auto allocSizeAndAlignment = 0x1u<<(core::findMSB(core::morton2d_encode_x(pageSize[0]-1u)|core::morton2d_encode_y(pageSize[1]-1u))+1);
```

Reply:
ok with latest ideas on how to handle mip-chains, the equation should be
```cpp
uint32_t rnd = core::roundUpToPoT((std::max(extent.width,extent.height)+m_pgSzxy-1u)/m_pgSzxy);
uint32_t pgCnt = rnd*rnd;
```